### PR TITLE
OWNERS for pipeline folder

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/967)

### CODE REVIEW

File: pipeline/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+  
+approvers:
+  - Bobgy
+  - IronPan
+  - rmgogogo
+reviewers:
+  - Bobgy
```
Comment (jlewi) [inline pipeline/OWNERS]: Reviewers shouldn't duplicate approvers.

Comment (Bobgy) [reply pipeline/OWNERS]: Thanks! Fixed

**Pull request merged**